docs/first-step/generate-native-tool/cli-make.rst

```diff
@@ -13,15 +13,15 @@ to the directory with sources:
   [cgold-example]> ls
   CMakeLists.txt foo.cpp
 
-Generate Makefile using CMake. Use :ref:`-H. <-H>` :ref:`-B_builds <-B>` for
+Generate Makefile using CMake. Use :ref:`-S . <-S>` :ref:`-B _builds <-B>` for
```

Comment:
Will not work with CMake 2.8, which is currently used in the first step:
- https://github.com/cgold-examples/minimal-example/blob/20d58555a7ffc703d06b386841bf3a72c754e672/CMakeLists.txt#L3